app/routes/provider.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+export default Ember.Route.extend({
+    theme: Ember.inject.service(),
+
+    providerIds: config.PREPRINTS.providers.map(provider => provider.id),
```

Comment:
Before, I had was using findAll in the `init` function on the route, but that created a race condition where the promise in `init` wouldn't resolve before `beforeModel` was called. Since we have to statically build the CSS anyway for each provider, it only makes sense to just add them into the config. This way, they're available immediately to the ember app instead of waiting for a promise to resolve (or error) if the network is slow. In phase 2, we're going to have to detect which domain we're on, and the only way to really do that to determine which routes to use is to have a config with the domains.